cfgov/v1/models/story_page.py

```diff
@@ -0,0 +1,47 @@
+from wagtail.admin.edit_handlers import (
+    ObjectList, StreamFieldPanel, TabbedInterface
+)
+from wagtail.core.blocks import StreamBlock
+from wagtail.core.fields import StreamField
+from wagtail.core.models import PageManager
+
+from v1.atomic_elements import molecules, organisms
+from v1.models.base import CFGOVPage
+
+
+class StoryHeader(StreamBlock):
+    jumbo_hero = molecules.JumboHero()
+    features = organisms.InfoUnitGroup()
+
+    class Meta:
+        block_counts = {
+            'jumbo_hero': {'max_num': 1},
+            'features': {'max_num': 1},
+        }
+
+
+class StoryContent(StreamBlock):
+    expandable_group = organisms.ExpandableGroup()
+    featured_content = organisms.FeaturedContent()
+    full_width_text = organisms.FullWidthText()
+    image = molecules.ContentImage()
+    info_unit_group = organisms.InfoUnitGroup()
+    text_introduction = molecules.TextIntroduction()
+    video_player = organisms.VideoPlayer()
+
+
+class StoryPage(CFGOVPage):
+    header = StreamField(StoryHeader, blank=True)
+    content = StreamField(StoryContent, blank=True)
```

Comment:
This is not important and not wrong as-is, but just as something you might be interested to know, `StoryHeader` (`CampaignHeader`, originally) was defined as its own class so that its `Meta` `block_counts` could be set. When that `Meta` is not needed for a StreamField, as in the case of the `content` StreamField here, we usually just define the StreamField items within `content = StreamField(…, blank=True)`.